src/main/java/fr/zcraft/quartzlib/tools/reflection/Reflection.java

```diff
@@ -94,6 +95,18 @@ public static String getMinecraftPackageName() {
         return Class.forName(getBukkitPackageName() + "." + name);
     }
 
+    /**
+     * Returns the {@link Class} of a NMS class from it's name (without the main NMS package) 1.17+.
+     * <p>For example, with "Server", this method returns the {@code net.minecraft.v1_X_RX} class.</p>
+     *
+     * @param name The NMS' class name (without the main Bukkit package).
+     * @return The class.
+     * @throws ClassNotFoundException if no class exists with this name in the NMS package.
+     */
+    public static Class getMinecraft1_17ClassByName(String name) throws ClassNotFoundException {
```

Comment:
The way minecraft 17 ordered classes is different, you will find them under net.minecraft then some will be in world.entity, nbt.nbttagcompound... In order to preserve previous code I have created a specific one for 1.17+